dali/frame.py

```diff
@@ -1,23 +1,12 @@
-from __future__ import unicode_literals
-from __future__ import division
-from __future__ import absolute_import
-from dali.compat import python_2_unicode_compatible
 import struct
-import sys
-
-
-if sys.version_info < (3,):
-    _integer_types = (int, long)
-else:
-    _integer_types = (int,)
 
+_integer_types = (int,)
```

Comment:
This is used for checking for numeric values with isinstance and has no benefit anymore. better check isinstance(foo, int) directly.

Reply:
Fair enough!